frame/evm/src/lib.rs

```diff
@@ -322,6 +328,36 @@ pub mod pallet {
 				pays_fee: Pays::No,
 			})
 		}
+
+		/// Increment `sufficients` for existing accounts having a nonzero `nonce` but zero `sufficients` value.
+		#[pallet::weight(
+			<T as pallet::Config>::WeightInfo::hotfix_inc_account_sufficients(addresses.len().try_into().unwrap_or(u32::MAX))
+		)]
+		pub fn hotfix_inc_account_sufficients(
```

Comment:
Can you put this under a feature flag?

Reply:
I tried it out but seems the generated code (`pallet::call` procedural macro) still contains references to this code even though the function body is omitted. Are you aware of a better way to put it behind a feature flag?

Reply:
Seems it's [discouraged](https://github.com/paritytech/substrate/issues/11151) to conditional compile pallet calls since that can break an API quite easily. We could conditionally compile and return an `unimplemented` error on invocation, but I'm unsure how useful that might be.

Reply:
Sorry to jump in like this:  
But should this not rather be a storage migration instead of an extrinsic?

Reply:
Yeah if you don't have too many accounts in the state already, a migration will be preferred. Alternatively, I believe you just need this call in one single runtime upgrade (which you manually call all addresses yourself), and then it can be removed. Given that this was a bug in Frontier that probably some other chains will also need it for fixing, we may include that in a Frontier release. We'd need to prepare a changelog and announce it though.